x-pack/plugins/observability_solution/infra/public/components/ml/anomaly_detection/anomalies_table/anomalies_table.tsx

```diff
@@ -74,11 +78,22 @@ const AnomalyActionMenu = ({
   influencerField: string;
   influencers: string[];
   disableShowInInventory?: boolean;
+  hostName?: string;
 }) => {
   const [isOpen, setIsOpen] = useState(false);
   const close = useCallback(() => setIsOpen(false), [setIsOpen]);
   const handleToggleMenu = useCallback(() => setIsOpen(!isOpen), [isOpen]);
   const { onViewChange } = useWaffleViewState();
+  const { metricsView } = useMetricsDataViewContext();
+  const {
+    services: {
+      share,
+      data: {
+        query: { filterManager: filterManagerService },
+      },
+    },
+  } = useKibanaContextForPlugin();
+  const hostLocator = share.url.locators.get('HOSTS_LOCATOR');
```

Comment:
We could type the `get` function and use constants for `HOSTS_LOCATOR`

```suggestion
  const hostLocator = share.url.locators.get<HostsLocatorDefinition>(HOSTS_LOCATOR_ID);
```

Reply:
As we spoke on Slack, I had build problem when using `HOSTS_LOCATOR_ID` constant. Thanks for help in solving this. Updated.